marketpulse/main/fxosprice_urls.py

```diff
@@ -4,8 +4,8 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^new/$', 'edit_fxosprice', name='edit_fxosprice'),
-    url(r'^all/$', 'all_fxosprice', name='all_fxosprice'),
-    url(r'^(?P<user_pk>\d+)/contributions/$', 'all_fxosprice', name='user_fxosprice'),
+    url(r'^contributions/all/$', 'view_contributions', name='all_contributions'),
```

Comment:
On top of that, ``list_all_contributions`` might be more descriptive but feel free to choose.